ibu-imager/seedcreator/seedcreator.go

```diff
@@ -500,19 +495,42 @@ func (s *SeedCreator) waitTillOvnKubeNodeIsDown(ctx context.Context) {
 	defer deadlineCancel()
 }
 
-func (s *SeedCreator) getCatalogImages(ctx context.Context) ([]string, error) {
+// filterCatalogImages filters catalog source images as catalog sources have pull always policy
+// and there is no point to precache them.
```

Comment:
Wait what?
The result of this PR is that the catalog source images will not be pre-cached?
Are you sure we want to do that?

Reply:
Yes